sortinghat/core/schema.py

```diff
@@ -77,6 +78,45 @@ def convert_json_field_to_generic_scalar(field, registry=None):
     return OperationArgsType(description=field.help_text, required=not field.null)
 
 
+def read_date_filter(filter_value):
+    """Extract the filter terms from a date filter
+
+    The accepted formats are controlled by regular expressions
+    matching two patterns: a comparison operator (>, >=, <, <=) and a date
+    OR a range operator (..) between two dates.
+
+    The accepted date format is ISO 8601, YYYY-MM-DDTHH:MM:SS
+
+    :param filter_value: String containing the filter value
+
+    :returns: A dictionary including an operator and the date values
+    """
+    # Accepted date format is ISO 8601, YYYY-MM-DDTHH:MM:SS
+    filter_data = {
+        "operator": None,
+        "date1": None,
+        "date2": None
+    }
+
+    # Filter with a comparison operator (>, >=, <, <=) and a date (e.g. `>=YYYY-MM-DDTHH:MM:SS`)
+    oper_comparison = r"^(<=?|>=?)(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})$"
```

Comment:
You are right, but I chose to follow this format because it is the one that other date fields are using (like the enrollment dates in the `Enroll` mutation), as it is the one coming from the `DateTime` type from `graphene`.